parts/linux/cloud-init/artifacts/mariner/cse_install_mariner.sh

```diff
@@ -31,7 +31,7 @@ installStandaloneContainerd() {
     # azure-built runtimes have a "+azure" suffix in their version strings (i.e 1.4.1+azure). remove that here.
     CURRENT_VERSION=$(containerd -version | cut -d " " -f 3 | sed 's|v||' | cut -d "+" -f 1)
     # v1.4.1 is our lowest supported version of containerd
-    local CONTAINERD_VERSION="1.4.3"
+    local CONTAINERD_VERSION="1.4.4"
```

Comment:
can we make this overridable from caller (i.e. RP) side? Just in case of CVE, we don't have to wait for CSE or VHD update

Reply:
we can address it later if we want to keep this PR simple

Reply:
yep, will address as a separate task as a part of our march engineering improvement effort